syncplay/messages_fi.py

```diff
@@ -10,93 +10,93 @@
 
      # Strings for Windows NSIS installer
      "installer-language-file": "Finnish.nlf", # Relevant .nlf file at https://github.com/kichik/nsis/tree/master/Contrib/Language%20files
-     "installer-associate": "Yhdistä Syncplay toimimaan multimedia-tiedostojen kanssa.",
-     "installer-shortcut": "Luo oikopolut seuraaviin kohteisiin:",
-     "installer-start-menu": "Käynnistysvalikko",
+     "installer-associate": "Aseta Syncplay avaamaan multimedia-tiedostoja.",
+     "installer-shortcut": "Luo pikakuvakkeet seuraaviin kohteisiin:",
+     "installer-start-menu": "Käynnistä-valokko",
```

Comment:
```suggestion
     "installer-start-menu": "Käynnistä-valikko",
```

Typoed Finnish term for start menu